### DISCUSSION TITLE
Unable to determine dependencies for Winter CMS. Your composer.json file is invalid.

### DISCUSSION THREAD

**Discussion by metismiao** [wintercms/winter#266] [Q&A] [answered]
While installing Winter, it shows

**Sorry, but an error has occurred while trying to install Winter CMS.
Unable to determine dependencies for Winter CMS. Your composer.json file is invalid.**

![螢幕擷取畫面 2021-08-04 175601](https://user-images.githubusercontent.com/40419327/128161684-3921b829-ad49-4253-8d32-78ee10875acf.jpg)

How can I solve this problem, thanks

**Comment by bennothommo** [comment]
Could you try making the changes from this commit (https://github.com/wintercms/web-installer/commit/ad670e1298c58947ac188a54339308123d373a58) to your installer and see if that makes a difference?

**Reply by metismiao** [reply]
I just tried (with 11 additions and 1 deletion), but it didn't work. 
still the same error message :(

**Reply by RomainMazB** [reply]
@metismiao can you confirm that following these guidelines make it work: https://github.com/wintercms/winter/discussions/251#discussioncomment-1051936

This problem seems to appear on slow connexion, is that your situation?

**Reply by metismiao** [reply]
Yes, there is a Status 500, but I did change the limit, still not working
I am using a shared hosting, I edited the .htaccess, not sure if it's working or not
after all, same error message as above.

![螢幕擷取畫面 2021-08-06 165411](https://user-images.githubusercontent.com/40419327/128485350-b9ac7739-7955-4da9-ac93-de0342a9004f.jpg)
![螢幕擷取畫面 2021-08-06 165724](https://user-images.githubusercontent.com/40419327/128485366-ccee766b-261d-4801-b7f8-a05f5ed88ab6.jpg)

**Comment by bennothommo** [comment, marked as answer]
@metismiao I've released a new beta release of the Installer, which contains several fixes and also includes logging. You can download it here: https://github.com/wintercms/web-installer/releases/download/v1.0.0-beta2/install.zip

If you have any further issues, please post a new issue to [Web Installer repo](https://github.com/wintercms/web-installer/issues/new) and include that log file - it might give us further info as to what is occurring.
